pom.xml

```diff
@@ -107,6 +107,9 @@
                 </executions>
                 <configuration>
                     <dotGitDirectory>${project.basedir}/.git</dotGitDirectory>
+                    <gitDescribe>
+                        <tags>true</tags>

```

Comment:
Please add a comment here in the XML explaining what this setting is for. I see in the PR description that this is a response to the fact that semantic release doesn't make annotated tags, but am not sure how this option resolves the problem.

Reply:
Might it be possible to instead get semantic release to create annotated tags? I think other tools including GitHub only consider annotated tags to be releases.

Reply:
Without further going a rabit hole I can't figure out how I'd make semantic release create releases with annotated tags.  I'll add a comment saying why this was added for now.